docker-for-windows/index.md

```diff
@@ -426,10 +428,10 @@ If the firewall rules appear to be open, consider [reinstalling the File and
 Print Sharing service on the virtual network
 adapter.](http://stackoverflow.com/questions/42203488/settings-to-windows-firewall-to-allow-docker-for-windows-to-share-drive/43904051#43904051)
 
-#### Shared drives on demand (Edge feature)
+#### Shared drives on demand
 
-On current Edge releases, sharing a drive can be done "on demand", the first
-time a particular mount is requested.
+Sharing a drive can be done "on demand", the first time a particular mount is
```

Comment:
You can share a drive "on demand" the first time a particular mount is requested.